web/js/map/wv.map.ui.js

```diff
@@ -875,9 +875,9 @@ wv.map.ui = wv.map.ui || function(models, config, components) {
       if (self.mapIsbeingDragged || wv.util.browser.small) {
         return;
       }
-      if (typeof models.naturalEvents == 'undefined' ||
-        typeof models.data == 'undefined' ||
-        models.naturalEvents.active || models.data.active) {
+      if ((typeof models.naturalEvents != 'undefined' &&
```

Comment:
This logic looks correct. It took a little bit of time for me to understand the original intention of the code, since there's not a lot of documentation, so let's take the opportunity to make things a little bit more readable here too. What do you think about doing something like this?

```
// Don't add data runners if we're on the events or data tabs
var isEventsTabActive = (typeof models.naturalEvents !== 'undefined' && models.naturalEvents.active);
var isDataTabActive = (typeof models.data !== 'undefined' && models.data.active);
if (isEventsTabActive || isDataTabActive) return;
```

@Benjaki2 does this look right to you too?

Reply:
Yes I think this would be much more readable, we could do the same for the map animation condition.